diff --git a/node_modules/@blueprintjs/core/src/components/menu/_submenu.scss b/node_modules/@blueprintjs/core/src/components/menu/_submenu.scss
index 151172b..27e6696 100644
--- a/node_modules/@blueprintjs/core/src/components/menu/_submenu.scss
+++ b/node_modules/@blueprintjs/core/src/components/menu/_submenu.scss
@@ -16,7 +16,7 @@
     &.#{$ns}-popover-open > .#{$ns}-menu-item {
       // keep a trail of hovered items as submenus are opened
       // stylelint-disable-next-line scss/at-extend-no-missing-placeholder
-      @extend .#{$ns}-menu-item:hover;
+      @extend .#{$ns}-menu-item, :hover;
     }
   }
 
diff --git a/node_modules/@blueprintjs/core/src/components/menu/_submenu.scss.rej b/node_modules/@blueprintjs/core/src/components/menu/_submenu.scss.rej
new file mode 100644
index 0000000..f5e909d
--- /dev/null
+++ b/node_modules/@blueprintjs/core/src/components/menu/_submenu.scss.rej
@@ -0,0 +1,17 @@
+***************
+*** 16,22 ****
+      &.#{$ns}-popover-open > .#{$ns}-menu-item {
+        // keep a trail of hovered items as submenus are opened
+        // stylelint-disable-next-line scss/at-extend-no-missing-placeholder
+-       @extend .#{$ns}-menu-item:hover;
+      }
+    }
+  
+--- 16,22 ----
+      &.#{$ns}-popover-open > .#{$ns}-menu-item {
+        // keep a trail of hovered items as submenus are opened
+        // stylelint-disable-next-line scss/at-extend-no-missing-placeholder
++       @extend .#{$ns}-menu-item, :hover;
+      }
+    }
+  
