diff --git a/server/node_modules/request/request.js b/server/node_modules/request/request.js
index 198b760..59f5d4d 100644
--- a/server/node_modules/request/request.js
+++ b/server/node_modules/request/request.js
@@ -288,13 +288,6 @@ Request.prototype.init = function (options) {
   if (!self.hasHeader('host')) {
     var hostHeaderName = self.originalHostHeaderName || 'host'
     self.setHeader(hostHeaderName, self.uri.host)
-    // Drop :port suffix from Host header if known protocol.
-    if (self.uri.port) {
-      if ((self.uri.port === '80' && self.uri.protocol === 'http:') ||
-          (self.uri.port === '443' && self.uri.protocol === 'https:')) {
-        self.setHeader(hostHeaderName, self.uri.hostname)
-      }
-    }
     self.setHost = true
   }
